src/__mocks__/grades.js

```diff
@@ -0,0 +1,88 @@
+import { v4 as uuid } from 'uuid';
```

Comment:
I do very much like this concept of committing mock data to be used while Firebase data store is being built out because it: 1. allows showing something in the UI, and 2. give the Firebase data store designer something to see as far as what the UI will be expecting